src/main/java/frc/robot/subsystems/shooter_intake/ShooterIntakeSubsystem.java

```diff
@@ -0,0 +1,145 @@
+package frc.robot.subsystems.shooter_intake;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.Logger;
+
+public class ShooterIntakeSubsystem extends SubsystemBase {
+
+    // Move to constants later
+    double intakeTargetRPM = -100.0;
+    double shootingTargetRPM = 300.0;
+    //
```

Comment:
Let's move these to constants

We can store constants in src/main/java/frc/robot/constants and eventually JSONify everything